app/models/user.rb

```diff
@@ -198,6 +198,10 @@ def in_person_pending_profile
     pending_profile if pending_profile&.in_person_verification_pending?
   end
 
+  def has_in_person_enrollment?
+    in_person_enrollments.order(created_at: :desc).first&.selected_location_details.present?
```

Comment:
A user may have multiple in-person enrollments. Usually we only want to consider the latest in-person enrollment, and only if that enrollment is in `:pending` status.

Considering any other in-person enrollment record will lead to unexpected behavior for users who run through this process multiple times.

Furthermore, a user may have an in-person enrollment record but then go back and use the remote process instead. The enrollment record will still exist for that case, but it may be marked as cancelled.

Reply:
Ok how about the `:establishing` status?  Would we also use that?

Reply:
The enrollment will be in `:establishing` status for the first few steps of the flow, which then transitions to `:pending` status shortly after the Instant Verify/AAMVA checks. A user should only have a single enrollment between `:establishing` and `:pending` status, though multiple enrollments may fall under other statuses. Past data may have enrollments that erroneously remain in `:establishing` or `:pending` status which would be more correctly marked as `:cancelled`.